Request/GetInboxMessagesRequestBuilder.php

```diff
@@ -0,0 +1,37 @@
+<?php
+namespace HP\Bundle\EmailApiBundle\Request;
+
+use HP\Bundle\EmailApiBundle\Request\DTO\GetInboxMessagesRequestDTO;
+
+/**
+ * Class GetInboxMessagesRequestBuilder
+ *
+ * Used to create a request (a dto) that will be sent to the usecase.
+ *
+ */
+class GetInboxMessagesRequestBuilder implements GetInboxMessagesRequestBuilderInterface
+{
+    /**
+     * @var GetInboxMessagesRequestDTO
+     */
+    private $dto;
+
+    public function create()
+    {
+        $this->dto = new GetInboxMessagesRequestDTO();
+
+        return $this;
+    }
+
+    public function setMaxResults($maxResults)
+    {
+        $this->dto->maxResults = $maxResults;
+
+        return $this;
+    }
+
+    public function build()
```

Comment:
True, the goal here was to make some sort of order like:
first create, then you set the parameters, then you build to return what you have just created.